packages/Webkul/Category/src/Repositories/CategoryRepository.php

```diff
@@ -286,9 +286,11 @@ private function setSameAttributeValueToAllLocale(array $data, ...$attributeName
             foreach (core()->getAllLocales() as $locale) {
                 foreach ($model->translatedAttributes as $attribute) {
```

Comment:
@papnoisanjeev 
we can put this check here to avoid unnecessary iterations for model->translatedAttributes.